pkg/ops/rawDiskGeneration.go

```diff
@@ -415,6 +418,48 @@ func (r *RawImage) createEFIPartitionImage() (string, error) {
 	return efiPartitionImage.File, nil
 }
 
+// createBiosPartitionImage creates a BIOS partition image with grub artifacts and installs grub to it
+func (r *RawImage) createBiosPartitionImage() (string, error) {
+	// Would need to format it and then copy the grub files to it
+	// Also install grub against it
+	// Remember to use the grub artifacts from the rootfs to support secureboot
```

Comment:
what secure boot? This is for bios booting, no?

Reply:
indeed, not sure if this was a comment brougth from the efi partition image function which was duped or soomething?